deploy/helm/edot-collector/kube-stack/values.yaml

```diff
@@ -0,0 +1,943 @@
+# For advanced configuration options, refer to the [official OpenTelemetry Helm chart](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-kube-stack/values.yaml)
+
```

Comment:
```suggestion
# This file has been tested together with opentelemetry-kube-stack helm chart version: 0.3.3
```

I find this comment super helpful in the values file, but we need to ensure we maintain this properly as soon as we decide to change the associated helm chart version ;)